### ISSUE TITLE
IGD: Force specific display output as primary / Multiply output

### ISSUE THREAD

**Issue by ghost** [Dasharo/dasharo-issues#835]
### The problem you're addressing (if any)

Recently i switched to the IGD as my primary graphics card. The issue is, if you use more than one display you can't set the 
 primary one, which for example would be helpful when using a monitor in portrait mode as otherwise it is really inconvenient to navigate settings in EDKII and/or Heads. The VBT shipping with Dasharo,Heads and also the original MSI BIOS uses HDMI as primary. Back when using Skylake and Coffee Lake,  i was able to set this myself using the Intel BMP tool to modify the VBT. But i couldn't find anything similar working for Alder Lake+. 

And as there is no support for Alder Lake+ in libgfxinit currently, there is no obvious way to fix this right now.
This only affects the boot process (BIOS settings), as the OS can set this itself.

### Describe the solution you'd like

1.) Give a option which display is primary
2.) Display the settings on all connected monitors

### Where is the value to a user, and who might that user be?

Making it easier navigating the settings during boot.
Everyone that uses the IGD with EDKII or Heads and has multiple monitors connected.

**Comment by mkopec** [comment]
Yeah, sadly the display configuration utility for newer platforms is NDA only. We would need to provide an alternate VBT I think
